### ISSUE TITLE
Web Interface 

### ISSUE THREAD

**Issue by B4erodriguez** [certtools/intelmq-manager#210]
Hello,

Maybe this is not an issue but I would like to know how to set up the intelmq-manager web interface, on which port does it listen?

I did the installation of intelmq, intelmq-manager and doing an "intelmqctl check" there are no errors found, but if I would like how to start the web interface since there is nothing that mentions it in the intelMQ-Manager installation documentation.

What is the command to start? do you have to perform a vhost in apache to run it? what are the steps?

Thank you.

**Comment by wagner-certat** [comment]
You followed https://github.com/certtools/intelmq-manager/blob/develop/docs/INSTALL.md? Then it's port 80.

**Comment by B4erodriguez** [comment]
Hi @wagner-certat !

Yes! I followed these steps as is as it appears there. The vhost in apache should be created alone or do you have to do it yourself? port 80 is open.

I am new to this tool and I had those doubts.

**Comment by wagner-certat** [comment]
If you used the repository, which branch did you use? master/maintenance or develop? Please don't use develop currently, we are switching the backend at the moment

**Comment by B4erodriguez** [comment]
I used master branch to install it as manually.

**Comment by wagner-certat** [comment]
Ok. In [this step](https://github.com/certtools/intelmq-manager/blob/master/docs/INSTALL.md#user-content-manually) you copied the files into the Document Root (`/var/www/html`), so they should be available via the default vhost. Can you check what is the default Document Root in your installation (if it's not `/var/www/html`)?

**Comment by B4erodriguez** [comment]
Yes! I copied the files into the Document Root (/var/www/html) and I see that: 

![image](https://user-images.githubusercontent.com/69397789/90517862-5c3cd780-e13c-11ea-8a63-c1fa01ef3cb4.png)

**Comment by wagner-certat** [comment]
That looks like the develop branch, not master. Please use only master for now, develop is not yet ready for production use. I will add some hints to the README and Installation documentation on this matter. Sorry for the circumstances.

**Comment by wagner-certat** [comment]
> I will add some hints to the README and Installation documentation on this matter. Sorry for the circumstances.

Done in 61549c4a8ed8bca87769e24fa3f85ada240febad

**Comment by wagner-certat** [comment]
Please let me know if there's something to improve in the Manger (including documentation)
